terrain/fixed_lod/voxel_terrain.cpp

```diff
@@ -377,8 +377,9 @@ void VoxelTerrain::set_block_enter_notification_enabled(bool enable) {
 
 	if (enable == false) {
 		const Vector3i *key = nullptr;
-		while ((key = _loading_blocks.next(key))) {
-			LoadingBlock *lb = _loading_blocks.getptr(*key);
+		
+		for (auto elm = _loading_blocks.begin(); elm < _loading_blocks.end(); ++elm) {
+			LoadingBlock *lb = _loading_blocks.getptr(elm->key);
```

Comment:
Can you use the iterator instead of doing a second hashmap lookup?

Reply:
Just pushed changes. Sorry did even noticed I did a redundant lookup.